stockflux-components/src/components/ScrollWrapperY/ScrollWrapperY.css

```diff
@@ -0,0 +1,14 @@
+.scroll-wrapper-y {
```

Comment:
Could this use CSS modules?

Reply:
As discussed today, not really as stockflux-components were not created using create-react-app.